vignettes/release-announcement.Rmd

```diff
@@ -0,0 +1,131 @@
+```{r}
+#| include: false
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're excited to announce that an initial version of the S7 package is now on CRAN.
+S7 is a new OOP system designed to be a successor to S3 and S4.
+The overarching goal of S7 is to combine the best parts of S3 and S4, so that the R community can eventually standardise on an single OOP system.
```

Comment:
```suggestion
The overarching goal of S7 is to combine the best parts of S3 and S4, so that the R community can eventually standardise on a single OOP system.
```